Makefile

```diff
@@ -11,6 +11,16 @@ test-minimal:
 	mkdir -p tests/logs
 	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) $(TEST_ARGS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
 
+test-minimal-tls: TEST_OUTFILE := tests/logs/test_minimal_out_$(shell date +%FT%T%Z).log
+test-minimal-tls:
+	mkdir -p tests/logs
+	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) --tags tls_adoption tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
```

Comment:
Do we need these targets? We can use the `TEST_ARGS` variable to provide `--tags`. Not a blocker for the initial merge though - if we want to get rid of these, it can be done afterwards.

Reply:
Right, I removed those